### PR TITLE
Fixed mobile version of Large-buttons

### PR DESCRIPTION
**Pull request by fabiennepimenta**
On mobile version we only can see the icon and on tablet/desktop
version we show the text.

### CODE REVIEW

File: build/index.html [hunk 1]
```diff
@@ -124,13 +125,13 @@
     <svg viewBox="0 0 100 100" class="icon">
       <use xlink:href="#symbol-add"></use>
     </svg>
-    <span>Add Booking</span>
+    <span class="text_btn">Add Booking</span>
```
Comment (menor) [inline build/index.html]: It's text inside a button so btn_text (please don't hate me)

File: src/styles/modules/_buttons.scss [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
+    padding: $large-button-padding--mobile;
     height: 100%;
     border-left: $large-buttons-border;
     border-right: $large-buttons-border;
     border-radius: 0;
-    margin-right: rem(-1);
+    margin-right: rem(-3);
     float: right;
-
+    .text_btn{
```
Comment (menor) [inline src/styles/modules/_buttons.scss]: We don't nest classes

File: src/styles/modules/_buttons.scss [hunk 3]
```diff
@@ -35,3 +37,13 @@
       color: $action-color-darker;
     }
   }
+
+
+@media screen and (min-width:720px){
```
Comment (menor) [inline src/styles/modules/_buttons.scss]: Let's put media queries inside the element, you can also use [@include media](http://thoughtbot.github.io/neat-docs/latest/#media) from neat

File: src/styles/modules/_header.scss [hunk 4]
```diff
@@ -2,5 +2,9 @@
   background: $pacific-blue;
   height: 60px;
   margin: 10px 0;
-  padding: $content-padding;
+  padding: 0;
+  width: 100%;
```
Comment (menor) [inline src/styles/modules/_header.scss:6]: Leave a blank line before a media query declaration

File: build/index.html (hunk 1 above)
Comment (fabiennepimenta) [inline build/index.html]: Lol... yeah....

**Pull request merged**